codemods/v2/borderedlist.ts

```diff
@@ -0,0 +1,65 @@
+/* eslint-disable no-param-reassign */
+import { FileInfo, API, Options } from 'jscodeshift';
+import { Codemod } from '../helpers';
+
+module.exports = function autocompleteProps(
+  fileInfo: FileInfo,
+  api: API,
+  options: Options,
+): string | null | undefined | void {
+  const mod = new Codemod(fileInfo, api);
+
+  mod.source.find(mod.cs.JSXElement).forEach(({ node }) => {
+    if (
+      node.openingElement.name.type === 'JSXIdentifier' &&
+      node.openingElement.name.name === 'BorderedList'
```

Comment:
You'll need to find the default import first since consumers can rename it to something else, for example.

```
import SomeList from '@airbnb/lunar/lib/components/BorderedList';
```

If they did rename, all these mods will fail. You can use `mod.getComponentNameFromDefaultImport()`.

Reply:
that's helpful! will update